packages/compiler-core/src/transforms/vFor.ts

```diff
@@ -308,7 +308,7 @@ export function processFor(
   }
 }
 
-const forAliasRE = /([\s\S]*?)\s+(?:in|of)\s+([\s\S]*)/
+const forAliasRE = /([\s\S]*?)\s+(?:in|of)\s+(\S[\s\S]*)/
```

Comment:
This change looks good to me, but this line has moved since this PR was first opened. It can now be found at:

https://github.com/vuejs/core/blob/94b9b37362310c34b846a634a5365b9b2d5db937/packages/compiler-core/src/utils.ts#L502